src/fvtt/page-script.js

```diff
@@ -25,9 +25,25 @@ class FVTTDisplayer {
 
     _postChatMessage(message, character, whisper, play_sound = false, attack_rolls, damage_rolls) {
         const MESSAGE_TYPES = CONST.CHAT_MESSAGE_TYPES || CHAT_MESSAGE_TYPES;
-        const pool = new DicePool([...attack_rolls, ...damage_rolls.map(d => d[1])].map(r => r._roll));
+        let pool;
+        pool = new DicePool([...attack_rolls, ...damage_rolls.map(d => d[1])].map(r => {
+            if (r instanceof FVTTRoll) { return r._roll; }
+            r.class = "Roll";
+            r.dice = [];
+            r.parts = r.parts.map(p => {
+                if (p.formula) {
```

Comment:
sounds good to me, alternative would be `if (typeof(p) === "string") return p;` but either way works. I don't have any strong preference actually.